packages/core/src/utils/crypto.ts

```diff
@@ -29,6 +29,10 @@ export class Crypto {
     const password = passwordBytes.toString("base64");
     return await this.storage().generateCryptoKey(password);
   }
+
+  async generateRandomKeyPair() {
```

Comment:
The keypair is already random so let's just call it
```suggestion
  async generateCryptoKeyPair() {
```